examples/chrono_binding.cpp

```diff
@@ -0,0 +1,173 @@
+#include <sqlite_orm/sqlite_orm.h>
+#include <cassert>
+#include <string>
+#include <iostream>
+#include <chrono>
+#include <format>
+
+#if __cpp_lib_chrono >= 201907L && __cpp_lib_format >= 202110L
```

Comment:
Do you need to test `__cpp_lib_format` against 202110? I.e. does your example depend on locale handling in chrono formatters and non-const-formattable types? Or can it just be the text formatting initially introduced in 201907?

Reply:
correct 201907 is enough